rpcs3/main.cpp

```diff
@@ -539,6 +539,9 @@ int main(int argc, char** argv)
 	sys_log.notice("Maximum open file descriptors: %i", utils::get_maxfiles());
 #endif
 
+	if (utils::get_low_power_mode())
```

Comment:
Missing braces

Reply:
They're also missing in the file limit code above.

Reply:
Then add them there as well. You're the author of that code.

Reply:
I didn't write the Linux code (which is also missing braces), but alright.